### DISCUSSION TITLE
Integrate Linux terminal solution like ttyd

### DISCUSSION THREAD

**Discussion by gauravsuman007** [DustinBrett/daedalOS#293] [Feature Requests]
This would make the OS quite useful, having Linux commands line tools at hand. 

At the very least, having ssh possibilities can enable using this as a remote terminal.

**Comment by DustinBrett** [comment]
Indeed that would be quite cool. Technically it's tricky without some kind of backend connection. I will look into what can be done with SSH over WebSockets, as that could support direct connections. Something like https://github.com/google/huproxy but ideally whichever is most popular and integrated in with an ssh server already would be great. That is how I am doing IRC via the new WebSockets spec in IRCv3.

I have added some Linux commands as aliases to the Windows versions, but not many Linux specific stuff yet as it's a bit more complex and usually involves piping or various other things that I don't have setup yet. There are projects like https://github.com/shelljs/shelljs which I am considering, but atm it's a bit of an undertaking.
